arm-siterecovery/2015-11-10/swagger/service.json

```diff
@@ -0,0 +1,13707 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2015-11-10",
+    "title": "SiteRecoveryManagementClient"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceNamespace}/{resourceType}/{resourceName}/replicationFabrics/{fabricName}/replicationvCenters/{vCenterName}/operationresults/{jobName}": {
+      "get": {
+        "tags": [
+          "VCenters"
+        ],
+        "summary": "Tracks the provider async operation.",
+        "description": "TODO (avrai): Detailed Description.",
```

Comment:
We are still working on documentation and examples part. We will update the PR once they are ready.